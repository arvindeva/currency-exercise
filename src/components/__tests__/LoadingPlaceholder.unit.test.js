import React from 'react';
import { shallow } from 'enzyme';

import LoadingPlaceholder from '../LoadingPlaceholder';

describe('Placeholder component', () => {
  test('renders Placeholder', () => {
    const wrapper = shallow(<LoadingPlaceholder />);
    expect(wrapper).toMatchSnapshot();
  });
});
