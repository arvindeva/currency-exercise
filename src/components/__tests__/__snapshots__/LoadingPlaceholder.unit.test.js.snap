// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder component renders Placeholder 1`] = `
<Placeholder
  fluid={true}
>
  <PlaceholderParagraph>
    <PlaceholderLine />
    <PlaceholderLine />
    <PlaceholderLine />
  </PlaceholderParagraph>
  <PlaceholderParagraph>
    <PlaceholderLine />
    <PlaceholderLine />
    <PlaceholderLine />
  </PlaceholderParagraph>
  <PlaceholderParagraph>
    <PlaceholderLine />
    <PlaceholderLine />
    <PlaceholderLine />
  </PlaceholderParagraph>
  <PlaceholderParagraph>
    <PlaceholderLine />
    <PlaceholderLine />
    <PlaceholderLine />
  </PlaceholderParagraph>
  <PlaceholderParagraph>
    <PlaceholderLine />
    <PlaceholderLine />
    <PlaceholderLine />
  </PlaceholderParagraph>
  <PlaceholderParagraph>
    <PlaceholderLine />
    <PlaceholderLine />
    <PlaceholderLine />
  </PlaceholderParagraph>
  <PlaceholderParagraph>
    <PlaceholderLine />
    <PlaceholderLine />
    <PlaceholderLine />
  </PlaceholderParagraph>
</Placeholder>
`;
